.github/CODEOWNERS

```diff
@@ -363,3 +363,5 @@
 /modules/services/volnoti.nix                         @IvanMalison
 
 Makefile                                              @thiagokokada
+
+/modules/services/screen-locker.nix                   @jrobsonchase
```

Comment:
While there is no precise ordering in the file, could you move this line to preserve some of the ordering?